nubis/bin/consul2unixusers.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pwd
+import ConfigParser
+import optparse
+import consul
+
+USERADD="/usr/sbin/useradd"
+CHOWN="/bin/chown"
+CHMOD="/bin/chmod"
+MKDIR="/bin/mkdir"
+DRYRUN=False
+
+_config = {}
+
+def load_config(config_file):
+    global _config
+    if not os.path.exists(config_file):
+        print "ERROR: Config file %s not found." % config_file
+        sys.exit(-1)
+    config = ConfigParser.RawConfigParser()
+    config.read(config_file)
+    if 'consul' not in config.sections():
```

Comment:
100% full config validation was not a goal I had for the first release of this tool. This is just a very low level sanity check, if other values are missing from the config file, Python's own errors will guide. The timeline I'm operating under doesn't allow for much else.